dev-client/src/screens/SlopeScreen/components/ManualSteepnessModal.tsx

```diff
@@ -139,6 +140,7 @@ export const ManualSteepnessModal = ({siteId}: Props) => {
                 helpText={t('slope.steepness.percentage_help')}
                 placeholder={t('slope.steepness.percentage_placeholder')}
                 textInputLabel={t('slope.steepness.percentage_placeholder')}
+                maxLength={3}
```

Comment:
Question: Is this redundant with the min/max specification above?

Reply:
Nope. This _prevents_ someone from typing in 9999. The min/max specification only shows an error afterwards.